data/hai/hai reveal 4 middle.txt

```diff
@@ -1868,6 +2026,17 @@ mission "Hai Reveal [A13-A] Unfettered Pickup"
 		fleet "Large Unfettered" 7
 
 
+mission "Hai Reveal [A13-B] Hai Election End Fix"
+	landing
+	invisible
+	to offer
+		has "Hai Reveal [A13] A New Elder: offered"
+		not "event: Hai election has ended"
+	on offer
+		event "Hai election has ended"
+
+
+
```

Comment:
This one is just in here so I don't have to replay Hai Reveal from the beginning to test the Wanderers Campaign. It can be removed before merge, but not yet.

Reply:
I have added a reminder in the PR description to remove this mission before merge.